libr/core/cmd_print.c

```diff
@@ -4030,23 +4052,25 @@ static int cmd_print(void *data, const char *input) {
 			ut32 n;
 			int i, c;
 			char buf[32];
-#define P(x) (IS_PRINTABLE(x)?x:'.')
-#define SPLIT_BITS(x) memmove (x + 5, x + 4, 5); x[4]=0
 			for (i = c = 0; i < len; i++,c++) {
 				if (c == 0) {
 					r_print_offset (core->print,
 						core->offset + i, 0, 0, 0, 0, NULL);
 				}
 				r_str_bits (buf, core->block+i, 8, NULL);
-				SPLIT_BITS (buf);
+
+				// split bits
+				memmove (buf + 5, buf + 4, 5);
+				buf[4] = 0;
+
 				r_cons_printf ("%s.%s  ", buf, buf+5);
 				if (c == 3) {
 					const ut8 *b = core->block + i-3;
-					#define K(x) (b[3-x]<<(8*x))
```

Comment:
Here I'd also keep the macho `K`, but modifying it to:

```c
 #define K(buf, x) (buf[3 - x] << (8 * x))
```

I'd also move it next to `P(x)`.